x-pack/legacy/plugins/ml/public/components/anomalies_table/severity_cell/severity_cell.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, memo } from 'react';
+import { EuiHealth } from '@elastic/eui';
+import { getSeverityColor } from '../../../../common/util/anomaly_utils';
+
+interface SeverityCellProps {
+  /**
+   * Severity score.
+   */
+  score: number;
+  /**
+   * Multi-bucket impact score from –5 to 5.
+   * Anomalies with a multi_bucket_impact value of greater than or equal
```

Comment:
I'd amend this comment to say something like `Anomalies with a multi-bucket impact value of greater than or equal to 2 are indicated with a plus shaped symbol in the cell`.